network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -1196,13 +1195,15 @@ def exec(self):
                                       'No traffic is specified as forbidden.')
         return IntersectsQuery(self.config1, self.config2).exec(True)
 
-    def compute_query_output(self, query_answer, _):
-        if query_answer.bool_result:
-            query_output = f'{self.config2.name} does not forbid connections specified in {self.config1.name}:' \
-                           f'{query_answer.output_explanation}'
+    def compute_query_output(self, query_answer, cmd_line_flag=False):
+        res = not query_answer.numerical_result if cmd_line_flag else query_answer.bool_result
+        query_output = query_answer.output_result + '\n'
+        if query_answer.bool_result and query_answer.numerical_result == 0:
+            query_output += f'{self.config2.name} does not forbid connections specified in {self.config1.name}: ' \
+                f'{query_answer.output_explanation}'
         else:
```

Comment:
I think this `else` should be replaced with `elif query_answer.numerical_result == 1` , to avoid adding this output when not relevant (as in my prev comment about not comparable/ no np cases.

Reply:
True.
Actually since the only case the numerical_result = 1 is when there is no intersection between the existing policies.
the `and query_answer.numerical_result == 0 `from line 1201 is not needed
and the  `elif query_answer.numerical_result == 1` is required here